src/features/views/components/ViewColumnDialog/SurveySubmitDateConfig.tsx

```diff
@@ -0,0 +1,77 @@
+import { useQuery } from 'react-query';
+import { useRouter } from 'next/router';
+import {
+  FormControl,
+  InputLabel,
+  MenuItem,
+  Select,
+  Typography,
+} from '@mui/material';
+import { FormattedMessage as Msg, useIntl } from 'react-intl';
+
+import getSurveys from 'features/smartSearch/fetching/getSurveys';
+import { COLUMN_TYPE, SelectedViewColumn } from '../types';
+
+interface SurveySubmitDateConfigProps {
+  onOutputConfigured: (columns: SelectedViewColumn[]) => void;
+}
+
+const SurveySubmitDateConfig = ({
+  onOutputConfigured,
+}: SurveySubmitDateConfigProps) => {
+  const intl = useIntl();
+
+  //TODO: Refactor to use model pattern
+  const { orgId } = useRouter().query;
+  const surveysQuery = useQuery(
+    ['surveysWithElements', orgId],
+    getSurveys(orgId as string)
+  );
+  const surveys = surveysQuery.data;
+
+  return (
+    <>
+      {!surveys || surveys.length > 0 ? (
```

Comment:
This is fine, but unnecessarily complex. Instead of returning a React fragment, where it's contents is deciding by "escaping" back to typescript (from TSX) and evaluating a ternary, the fragment and "escape" can be omitted:

```tsx
return !survey || surveys.length > 0 ? (
  <FormControl…
```

The fragment is only necessary to return two or more children.
```

Reply:
durgh ofcourse, I thought when I wrote it that something was not looking right... I want to change it so I'm gonna :)